lib/hooks/userspace/hook-express.js

```diff
@@ -106,28 +105,26 @@ function endRootSpanForRequest(rootContext, req, res) {
   rootContext.close();
 }
 
-module.exports = {
-  patch: function(express_, agent_, version_) {
-    if (!semver.satisfies(version_, SUPPORTED_VERSIONS)) {
-      agent_.logger.info('Express: unsupported version ' + version_ + ' loaded');
-      return;
-    }
-    if (!express) {
-      agent = agent_;
-      express = express_;
-      patchedMethods.forEach(function(method) {
-        shimmer.wrap(express.application, method, applicationActionWrap);
-      });
-    }
-  },
-  unpatch: function() {
-    if (express) {
-      patchedMethods.forEach(function(method) {
-        shimmer.unwrap(express.application, method);
-      });
-      agent.logger.info('Express: unpatched');
-      express = null;
-      agent = null;
-    }
+module.exports = function(version_, agent_) {
+  if (!semver.satisfies(version_, SUPPORTED_VERSIONS)) {
+    agent_.logger.info('Express: unsupported version ' + version_ + ' loaded');
+    return {};
   }
+  return {
+    'index.js': {
```

Comment:
I have changed these relative file paths to be the empty string in the case where we want to patch the module itself. I've added documentation describing the semantics of the empty key here.